grafanalib/cloudwatch.py

```diff
@@ -26,7 +26,8 @@ class CloudwatchMetricsTarget(object):
     :param period: Cloudwatch data period
     :param refId: target reference id
     :param region: Cloudwatch region
-    :param statistics: Cloudwatch mathematic statistic
+    :param statistics: Cloudwatch mathematic statistics
```

Comment:
Unsolicited advise: Should there be a message e.g. `(to be deprecated, prefer `statistic` instead)`